main.cpp

```diff
@@ -16,11 +17,22 @@ int main(int argc, char *argv[]) {
       break;
     }
   }
-  QFrame q;
+
+  MainWindow main;
   MockModel m;
-  Home h(&q, &m);
+  QFrame *mainFrame = new QFrame(&main);
+  Home *h = new Home(mainFrame, &m);
+  h->setFrameShape(QFrame::Box);
+  h->setLineWidth(2);
+
+  QVBoxLayout *layout = new QVBoxLayout;
+  layout->addWidget(h);
+
+  QWidget *centralWidget = new QWidget(&main);
+  centralWidget->setLayout(layout);
 
-  h.show();
+  main.setCentralWidget(centralWidget);
+  main.show();
```

Comment:
@Peyton-McKee can u send me the documentation u looked at to setup this file. Some of the reference/pointer shit just looks really weird, but maybe this is j how it's done in qt

Reply:
I sent u it in slack

Reply:
Oh wait u mean the main file